src/neuroglancer/segmentation_user_layer.ts

```diff
@@ -482,9 +492,14 @@ export class SegmentationUserLayer extends Base {
             loadedSubsource.activate(refCounted => {
               this.graphConnection = refCounted.registerDisposer(
                   segmentationGraph.connect(this.displayState.segmentationGroupState.value));
-              refCounted.registerDisposer(() => {
-                this.graphConnection = undefined;
-              });
+              const segmentationRenderlayer = this.someSegmentationRenderLayer();
```

Comment:
`someSegmentationRenderLayer` isn't the right approach, because if another non-graphene data source has been added to the same layer it may find the render layer from that.

Additionally, if the segmentation volume has been manually unchecked by the user, there won't be a segmentation layer either.

Is there a reason why the graph connection can't just supply everything it needs directly?

Reply:
Thinking about this more, I see that there is a need to have some dummy renderlayer that will be picked up by the SliceView.